llvm/test/CodeGen/RISCV/cm_mvas_mvsa.ll

```diff
@@ -163,3 +167,12 @@ define i32 @not_zcmp_mv(i32 %num, i32 %f) nounwind {
   %res = call i32 @func(i32 1, i32 %f)
   ret i32 %res
 }
+
+; CHECK32ZCMPMIR-LABEL: name: zcmp_mv
+; CHECK32ZCMPMIR: $x9, $x8 = CM_MVSA01 implicit $x10, implicit $x11
+; CHECK32ZCMPMIR: CM_MVA01S killed $x9, $x8, implicit-def $x10, implicit-def $x11
+; CHECK32ZCMPMIR-LABEL: name: not_zcmp_mv
+; CHECK32ZCMPMIR-NOT: CM_MVSA01
+; CHECK32ZCMPMIR-NOT: CM_MVA01S
+;; NOTE: These prefixes are unused and the list is autogenerated. Do not add tests below this line:
+; CHECK32ZCMPMIR: {{.*}}
```

Comment:
Is CHECK32ZCMPMIR unused? It looks like there are test check lines for it 171-176.

Reply:
It's not, but `update_llc_tests_checks` thinks it is. I wanted to avoid splitting this into another separate MIR test.

Reply:
do you know why update_llc_tests_checks is generating this unused label despite it also using this label directly above? I have never seen this before.

Reply:
probably something to do with those lines being generated by update_mir_test_checks.py rather than update_llc_test_checks.py.

Reply:
I think you should probably move this into a separate MIR test:
* The RUN line is generating MIR and the other tests are generating ASM
* The existing tests are using update_llc_test_checks and you are writing manual checks
* update_llc_test_checks is the wrong script to use if you were auto-generating tests for this RUN line
* The IR is small